components/questionnaire/QuestionBox.tsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from "react";
+import { QuestionProperties } from "../../interfaces/Question";
+import { QuestionWithAnswear } from "../../interfaces/QuestionWithAnswear";
+import { BigButton } from "./BigButton";
+import { Question } from "./Question";
+
+export function QuestionBox() {
+  const answears = [1, 2, 3, 4, 5];
+
+  const [questions, setQuestions] = useState<QuestionProperties[]>([
+    {
+      question:
+        "Jak podobał Ci się nasz staż? (ostatnia szansa na zmianę zdania! :D)",
+      hint: "(1 - w ogóle, 2 - niezbyt, 3 - średnio, 4 - był ok, 5 - bardzo)",
+      key: Math.random(),
+      selected: 0,
+    },
+    {
+      question: "Jak dobrze był Twoim zdaniem przygotowany prowadzący?",
+      hint: "(1 - fatalnie, 2 - kiepsko, 3 - średnio, 4 - był ok, 5 - świetnie)",
+      key: Math.random(),
+      selected: 0,
+    },
+    {
+      question: "Czy uważasz, że zdobyta wiedza przyda Ci się w życiu?",
+      hint: "(1 - w ogóle mi się nie przyda, 2 - raczej nie, 3 - może, 4 - myślę, że tak, 5 - na pewno!)",
+      key: Math.random(),
+      selected: 0,
+    },
+  ]);
```

Comment:
For readability, we could move the static content (questions & hints) to a separate file and import it, and keep only the changing values in the state